src/TokenHandler/AccountManager.mo

```diff
@@ -82,6 +86,7 @@ module {
       if (fee_ != newFee) {
         journal.push((Time.now(), ownPrincipal, #feeUpdated({ old = fee_; new = newFee })));
         recalculateBacklog(fee_, newFee);
+        recalculateDust();
```

Comment:
Can we call only one of them, recalculateBacklog or recalculateDust, based on whether the fee went up or down?

Reply:
While recalculating one of them we can still push to the other. But it means we are looping over the elements of only one of them.

Reply:
Thanks, fixed